README.md

```diff
@@ -1,224 +1,227 @@
+
+It is recommended to use [IntelliJ](https://www.jetbrains.com/idea/).
+Install the Java17 JDK ([Temurin](https://adoptium.net/) is the recomended distribution).
+Then, [clone](https://www.jetbrains.com/help/idea/set-up-a-git-repository.html#clone-repo) the Chunky
+repository and let IntelliJ index the project. Navigate to `chunky/src/java/se/llbit/chunky/main/Chunky.java` and 
+click on the green play button next to `public class Chunky {` to build and run Chunky.
+
+To build Chunky externally, run the `gradlew` script in the project root directory. Gradle is setup with a few
+main tasks:
+* `build` - Build Chunky, documentation, and run tests.
+* `release` - Build and save files ready for a release to a Chunky update site. Outputs to `build/release`
+* `buildReleaseJar` - Build an installer JAR. Outputs to `build/installer`
+* `docs` - Build the documentation. Outputs to `build/docs`
+* `install` - Create a publishable maven repository for Chunky core. Outputs to `build/maven`
+* `clean` - Cleans the project. Removes old builds.
+
+A custom version can be specified with `-PnewVersion="<version>"`. A custom prerelease tag can be specified with
+`-PprereleaseTag="<tag>"`. The default version is in the format: `{major}.{minor}.{patch}-{tag (DEV)}.{commits since last tag}.g{git hash of commit}`
+
+Chunky is split into four subprojects:
+
+* **chunky** - the core rendering and GUI project
+* **lib** - common code required by the other projects
+* **launcher** - the launcher
+* **releasetools** - tool used for packaging releases
+
+If you want to hack on Chunky itself you will need to load the `chunky` and
+`lib` directories in your favorite editor. If available, use a Gradle project
+import option.
+
+
+### Code Style
+
+The [Google Java style guide][Google-styleguide] should be followed for new code (2 spaces for
+indentation, no tabs). If you want to contribute code to Chunky please make
+your code look similar to the rest of the code, and refer to the style guide
+when in doubt.
+
+
+## Copyright & License
+
+Chunky is Copyright (c) 2010-2022, Jesper Öqvist <jesper@llbit.se> and [Chunky Contributors][chunky-contributors]. 
+
+Permission to modify and redistribute is granted under the terms of
+the GPLv3 license. See the file `LICENSE` for the full license.
+
+Chunky uses the following 3rd party libraries:
+
+* **Apache Commons Math library by the Apache Software Foundation.**
+  The library is covered by the Apache License, version 2.0.
+See the file `licenses/Apache-2.0.txt` for the full license text.
+See the file `licenses/commons-math.txt` for the copyright notices.
+* **FastUtil by Sebastiano Vigna**.
+  FastUtil is covered by Apache License, version 2.0. 
+See the file `licenses/Apache-2.0.txt` for the full license text.
+See the file `licenses/fast-util.txt` for the copyright notice.
+* **Gson by Google**
+  The library is convered by the Apache License, version 2.0.
+See the file `licenses/Apache-2.0.txt` for the full license text.
```

Comment:
We should add a `licenses/gson.txt` file that contains the copyright notice, ie. this block of text: https://github.com/google/gson#license